src/common/wallet/basic.wallet.js

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+
+const ordinal = require('ordinal');
+
+export default class BasicWallet {
+  constructor(id, name, walletType) {
+    this.coins = [];
+    this.id = id;
+    this.name = name || `My ${ordinal(id + 1)} Wallet`;
+    this.walletType = walletType;
+  }
+
+  // create coins and add to list
+  createCoins = (coins) => {
+    if (!_.isEmpty(coins)) {
+      coins.forEach((item) => {
+        this.addToken(item);
+      });
+    }
```

Comment:
Using _.each(coins) you don't need to check !_.isEmpty()

_.each() will not throw error if coins is not defined.

Reply:
Fixed